benchmark/realnest/16_most_common_list_aggregates.benchmark

```diff
@@ -0,0 +1,82 @@
+# name: benchmark/realnest/16_most_common_list_aggregates.benchmark
+# description: Combination of the most common list_aggregate functions
+# group: [realnest]
+
+name most_common_list_aggregates
+group real_nest
+
+require json
+
+require httpfs
+
+cache real_nest.duckdb
+
+load benchmark/realnest/load.sql
+
+run
+SELECT 
+  list_aggregate(tau_pt, 'list'),
```

Comment:
Also in the other benchmarks, you can rewrite some of the `list_aggregates` to their respective rewrites.
`list`, `list_sum`, ... What you prefer. 😄